tests/Microsoft.VisualStudio.ProjectSystem.Managed.UnitTests/Utilities/SynchronizationContextUtil.cs

```diff
@@ -0,0 +1,28 @@
+﻿// Licensed to the .NET Foundation under one or more agreements. The .NET Foundation licenses this file to you under the MIT license. See the LICENSE.md file in the project root for more information.
+
+namespace Microsoft.VisualStudio;
+
+internal static class SynchronizationContextUtil
```

Comment:
This could really use a doc comment explaining what it is, why we need it, and, if possible, guidance on when to use it.

Reply:
I added a comment to the `Suppress` method that provides detail as to why I added this. Let me know if it's unclear or could be improved.